jsonld/examples/package_sbom.json

```diff
@@ -0,0 +1,79 @@
+{
+    "@context": "https://raw.githubusercontent.com/spdx/spdx-3-serialization-prototype-playground/main/jsonld/spdx-3.0-context.json-ld",
+    "@graph": [
+        {
+            "type": "CreationInfo",
+            "spdxId": "_:creationinfo",
+            "createdBy": "http://spdx.example.com/Agent/JoshuaWatt",
+            "specVersion": "3.0.0",
+            "created": "2024-03-06T00:00:00Z"
+        },
+        {
+            "type": "Person",
+            "spdxId": "http://spdx.example.com/Agent/JoshuaWatt",
+            "name": "Joshua Watt",
+            "creationInfo": "_:creationinfo",
+            "externalIdentifier": [
+                {
+                    "type": "ExternalIdentifier",
+                    "externalIdentifierType": "email",
+                    "identifier": "JPEWhacker@gmail.com"
+                }
+            ]
+        },
+        {
+            "type": "SpdxDocument",
+            "spdxId": "http://spdx.example.com/Document1",
+            "creationInfo": "_:creationinfo",
+            "rootElement": [
+                "http://spdx.example.com/BOM1"
+            ],
+            "profileConformance": [
+                "core",
+                "software"
+            ]
+        },
+        {
+            "type": "software_Sbom",
+            "spdxId": "http://spdx.example.com/BOM1",
+            "creationInfo": "_:creationinfo",
+            "rootElement": [
+                "http://spdx.example.com/Package1"
+            ],
+            "software_sbomType": "build"
+        },
+        {
+            "type": "software_Package",
+            "spdxId": "http://spdx.example.com/Package1",
+            "creationInfo": "_:creationinfo",
+            "name": "my-package",
+            "software_packageVersion": "1.0",
+            "software_downloadLocation": "http://dl.example.com/my-package_1.0.0.tar",
+            "builtTime": "2024-03-06T00:00:00Z",
+            "originatedBy": "http://spdx.example.com/Agent/JoshuaWatt"
+        },
+        {
+            "type": "software_File",
+            "spdxId": "http://spdx.example.com/Package1/myprogram",
+            "creationInfo": "_:creationinfo",
+            "name": "myprogram",
+            "software_primaryPurpose": "executable",
+            "software_additionalPurpose": [
+                "application"
+            ],
+            "software_copyrightText": "Copyright 2024, Joshua Watt",
+            "builtTime": "2024-03-06T00:00:00Z",
+            "originatedBy": "http://spdx.example.com/Agent/JoshuaWatt"
+        },
+        {
+            "type": "Relationship",
```

Comment:
`Relationship` should have an `spdxId` -- is is an Element

Reply:
Will do